crates/uv-distribution/src/pyproject_mut.rs

```diff
@@ -424,14 +424,36 @@ fn reformat_array_multiline(deps: &mut Array) {
             })
     }
 
+    let mut indentation_prefix = None;
+
     for item in deps.iter_mut() {
         let decor = item.decor_mut();
         let mut prefix = String::new();
+        // calculating the indentation prefix as the indentation of the first dependency entry
+        if indentation_prefix.is_none() {
+            let decor_prefix = decor
+                .prefix()
+                .and_then(|s| s.as_str())
+                .map(|s| s.split('#').next().unwrap_or("").to_string())
```

Comment:
Hmm I'm a little unclear how this works if the line doesn't have a comment?

Reply:
If the `decor.prefix()` does not have a comment this whole expression will return the `String` that is in `decor.prefix()` trimmed by `"\n"` from the left

Reply:
If you're asking about this particular line
```
.map(|s| s.split('#').next().unwrap_or("").to_string())
```
in case there is no '#' in the string it well return the whole string

Reply:
Ah that makes sense.